component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/reporter/cloud/DefaultInputValidator.java

```diff
@@ -75,6 +76,7 @@ public class DefaultInputValidator {
             new AbstractMap.SimpleImmutableEntry<>(API_VERSION, String.class),
             new AbstractMap.SimpleImmutableEntry<>(API_CREATION, String.class),
             new AbstractMap.SimpleImmutableEntry<>(API_METHOD, String.class),
+            new AbstractMap.SimpleImmutableEntry<>(API_CONTEXT, String.class),
```

Comment:
Did we make the change from synapse side to send API context

Reply:
Yes we did Tishan.